src/govuk/helpers/colour.test.js

```diff
@@ -35,24 +35,38 @@ describe('@function govuk-colour', () => {
 
       .foo {
         color: govuk-colour('red');
-      }`
-
-    const results = await renderSass({ data: sass, ...sassConfig })
+      }
+    `
 
-    expect(results.css.toString().trim()).toBe('.foo { color: #ff0000; }')
+    await expect(renderSassString(sass, sassConfig))
+      .resolves
+      .toMatchObject({
+        css: outdent`
+          .foo {
+            color: #ff0000;
+          }
+        `
+      })
   })
 
-  it('works with unquoted strings', async () => {
+  it.skip('works with unquoted strings', async () => {
```

Comment:
This test is great for LibSass (and Ruby Sass) but strings must now be quoted in Dart Sass it seems

Reply:
Fixed in:

* https://github.com/alphagov/govuk-frontend/pull/3192